core/case_study/views.py

```diff
@@ -307,3 +308,44 @@ def submit_comment(request, case_study_id):
         }
     }
     return JsonResponse(data)
+
+@csrf_exempt
```

Comment:
Please look up what csrf is before you disable it
https://docs.djangoproject.com/en/2.2/ref/csrf/

This should be @login_required

You can fix this by using `fetch` and adding this to your headers `"X-CSRFToken": getCookie("csrftoken"),`
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API